jobs/job_utils.py

```diff
@@ -55,27 +55,70 @@ def clone_model(model, **new_values):
     return cls(id=model_id, **props)
 
 
-def get_model_kind(item):
-    """Returns the "kind", a globally unique identifier, of the given item.
+def get_model_kind(model):
+    """Returns the "kind" of the given model.
 
     NOTE: A model's kind is usually, but not always, the same as a model's class
-    name. This function will always return the correct value for "kind", even if
-    it is different from the class's name.
+    name. This function will always return the correct value for "kind", even
+    when it is different from the class's name.
```

Comment:
Maybe explain a circumstance in which it is different.

Reply:
Done.